articles/npm-packages/index.md

```diff
@@ -0,0 +1,191 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/npm-packages/
+title: Create and Deploy NPM Packages
+description: Introduction to node modules and a hands-on about how to create and deploy a node module to the npm registry.
+author: saiharsha-balasubramaniam
+date: 2020-09-23T00:00:00-12:00
+topics: [Node.js]
+excerpt_separator: <!--more-->
+images:
+  - url: /engineering-education/npm-packages/hero.jpg
+    alt: NPM
+---
+
+Node.js is a popular server-side language that is used by millions of developers worldwide. While building Node.js projects, we use various packages that make development easier. All of these packages are managed by the Node Package Manager.
+
+<!--more-->
+
+### Table of Contents
+
+- [Introduction](#introduction)
+- [Create an NPM Account](#create-an-npm-account)
+- [Let's Code](#let's-code)
+- [Deployment](#deployment)
+- [Using the Package](#using-the-package)
+- [Further Reading](#further-reading)
+
+### Introduction
+
+A package manager is a software that manages versions, packages and dependencies for your project. When a project grows in scale, a package manager helps to manage dependencies. It works perfectly across versions and even in collaborative projects. It cuts costs by reducing dependency problems across developer teams.
+
+To install Node.js packages, we use the [NPM registry](https://www.npmjs.com/), which is a centralized repository of packages. In this article we will learn how to create a Node.js package and deploy it on the NPM registry for public use.
+
+For more background on Node.js and various related technologies, check out some amazing [Section.io](https://www.section.io/) articles below:
+
+- [History of Node.js](/engineering-education/history-of-nodejs/)
+- [Most Useful Node.js Packages](/engineering-education/most-useful-nodejs-packages/)
+- [Building a Node.js Application Using Docker](/engineering-education/building-a-nodejs-application-using-docker/)
+- [Linting in Node.js using ESLint](/engineering-education/node-eslint/)
+- [APIs in Node.js vs Python - A Comparison](/engineering-education/node-vs-python/)
+
+### Create an NPM Account
+
+To start up, let us create an NPM account.
+
+- Go to the [npmjs website](https://www.npmjs.com/) and click on the **sign up** button.
+
+![npmjs Webpage](/engineering-education/npm-packages/npmjs-homepage.png)
+
+- Enter a username, email address and password.
+
+![Create NPM Account](/engineering-education/npm-packages/npm-signup.png)
+
+- Now, you have to verify your account. Check your email ID that you provided and click on the verification email. This will verify your npm account.
```

Comment:
Check the inbox of the email address you provided would make more sense and flow better.